templates/docker-compose.yml

```diff
@@ -112,3 +115,29 @@ services:
 {% endif %}
     restart: "always"
     logging: *default-logging
+
+{% if pictrs_safety is defined and pictrs_safety|bool == true %}
+  pictrs-safety:
+    image: ghcr.io/db0/pictrs-safety:v1.2.2
+    hostname: "pictrs-safety"
```

Comment:
you don't need quotes here

Reply:
They don't hurt though. If you insist on the aesthetics I can remove them